Src/Scripting/NativeDefinitions/INIDefinitions.h

```diff
@@ -0,0 +1,18 @@
+#ifndef INIDEFINITIONS_H_INCLUDED
+#define INIDEFINITIONS_H_INCLUDED
+
+#include <vector>
+#include <map>
+#include <Misc/FilePath.h>
+#include "../../Utils/INI.h"
+#include "../NativeDefinition.h"
+
+
+class ScriptManager;
+
+class INIDefinitions : public NativeDefinition {
+public:
+    INIDefinitions(ScriptManager* mgr, INIFile* ini);
+};
+
+#endif // INIDEFINITIONS_H_INCLUDED
```

Comment:
Don't forget the trailing newline.

Reply:
Sorted.
I'm fairly certain there's a couple of other places where I have done this.

I believe this file is generally finalized.
Whether or not it ought to be used is a different matter.
The INI file functions are all already there, so may as well have the angelscript definitions.
If nothing more, i'm thinking it can be used to parse the previous ini files and turn them into a new format, then fix the errors. Saves having to do all of that manually.
I'll give it a test before resolving this convo.